pb/threads_pb.d.ts

```diff
@@ -0,0 +1,480 @@
+// package: threads.pb
```

Comment:
Are we sure we want to check these into version control? Maybe good for now, but might be better to just export and publish as a package at some point right?

Reply:
Totally. Let's see how these custom gogo types play with JS and go from there... I can then pull over the npm make-and-publish-on-tag stuff from `go-textile`